File: src/main/java/no/nav/veilarboppfolging/client/ytelseskontrakt/VedtakDto.java
package no.nav.veilarboppfolging.client.ytelseskontrakt;

import lombok.Data;
import lombok.experimental.Accessors;

@Data
@Accessors(chain = true)
public class VedtakDto {
    String vedtakstype;
    String status;
    String aktivitetsfase;
    String rettighetsgruppe;
    Dato fradato;
    Dato tildato;
}

File: src/main/kotlin/no/nav/veilarboppfolging/client/ytelseskontrakt/VedtakDto.kt
package no.nav.veilarboppfolging.client.ytelseskontrakt

data class VedtakDto(
    val vedtakstype: String?,
    val status: String?,
    val aktivitetsfase: String?,
    val rettighetsgruppe: String?,
    val fradato: Dato?,
    val tildato: Dato?,
)
